### DISCUSSION TITLE
Concerns Regarding MDN JavaScript Icon Change

### DISCUSSION THREAD

**Discussion by mdmuhtasimfuadfahim** [mdn/mdn-community#686] [Announcements]
I am writing to express my concern about MDN's recent decision to change the JavaScript icon to promote LGBTQ+ awareness. While supporting the rights and dignity of all individuals is important, I believe that a responsible organization should remain neutral and avoid promoting specific communities.

MDN is a critical resource for developers worldwide, encompassing diverse cultural and religious backgrounds. Promoting specific social or political causes, even with good intentions, can create conflicts and alienate users, including those from Muslim and other religious communities who may hold different views.

I respectfully request that MDN reconsider this icon change to maintain a neutral stance. This approach will help ensure that MDN remains an inclusive and welcoming platform for all users, regardless of their personal beliefs.

Thank you for considering this feedback.

**Comment by HerminaC** [comment]
Hi @mdmuhtasimfuadfahim,

Thank you for sharing your concerns. At MDN, we are committed to promoting an inclusive environment that welcomes everyone. The update to the icon is a reflection of our ongoing commitment to diversity and inclusion, which are integral to Mozilla’s core values. While we appreciate your feedback and consider the diverse perspectives of our global community, we will maintain the current icon as is. Our primary focus continues to be providing high-quality, reliable documentation for developers around the world. We value your feedback and I thank you for being a part of the MDN community.

Regards,
Hermina

**Reply by mdmuhtasimfuadfahim** [reply]
Hi @HerminaC,

Thank you for your thoughtful response. However, I would like to express that if promoting diversity hurts other communities, can it truly be considered inclusive? As a Muslim, I find it difficult to support this change. I appreciate your courteous reply and commitment to inclusivity, but if your actions alienate a significant community, it challenges the very notion of inclusiveness.

Best,
Muhtasim

**Reply by HerminaC** [reply]
I understand we may see things differently. Affirming our support for the LGBTIQA+ community through the icon update is intended to promote inclusiveness and is not meant to alienate or disrespect any group—I am sorry you feel this way. We will maintain our decision in this instance but I hope you will continue to find value in MDN as a resource.

Kindly,
Hermina
